plugins/linux-v4l2/v4l2-udev.c

```diff
@@ -157,6 +164,10 @@ void v4l2_init_udev(void)
 	if (udev_refs == 0) {
 		if (os_event_init(&udev_event, OS_EVENT_TYPE_MANUAL) != 0)
 			goto fail;
+		if ((udev_event_fd = eventfd(0, EFD_CLOEXEC)) < 0) {
```

Comment:
I wonder if `EFD_NONBLOCK` shouldn't be used here too?

Reply:
I'm not sure it makes sense, we will never be in a blocking case since we only write to the fd once and we want that write to succeed. If it were nonblocking it could fail instead of blocking which would be unfortunate. Maybe its ok to make it nonblocking if we retain the timeout on poll.